Instructor's notes.md

```diff
@@ -20,16 +20,24 @@
 
 - #### linux_file_system
 
-  Most of information are taken from "Introduction to Linux" and "Linux bible" books and following sites: https://www.linuxnix.com/linux-directory-structure-explained-boot-folder/ 
+  Most of information are taken from "Introduction to Linux" and "Linux bible" books and following sites: 
+  
+  [first site](https://www.linuxnix.com/linux-directory-structure-explained-boot-folder/)
```

Comment:
Please, use a better name (Maybe the name of the website itself) for referencing.